### ISSUE TITLE
Resulted 2D pose of the mentioned HR-Net on Human 3.6M 

### ISSUE THREAD

**Issue by wbhu** [Nicholasli1995/EvoSkeleton#26]
Hi,

Thanks for your great work and code. I noticed that you released the pre-trained model of HR-Net for 2D pose estimator, but we still need the crop step, which may lead to an inconsistent 2D result. Could you please directly provide the estimated 2D pose (it may be in .npy format and shouldn't be too large) from the HR-Net on Human 3.6M? I think it will be very helpful for the community.

**Comment by Nicholasli1995** [comment]
> Hi,
> 
> Thanks for your great work and code. I noticed that you released the pre-trained model of HR-Net for 2D pose estimator, but we still need the crop step, which may lead to an inconsistent 2D result. Could you please directly provide the estimated 2D pose (it may be in .npy format and shouldn't be too large) from the HR-Net on Human 3.6M? I think it will be very helpful for the community.

Hi, thank you for the interest in our work. The 2D pose files have already been released. Please go to https://github.com/Nicholasli1995/EvoSkeleton/blob/master/docs/TRAINING.md and you can download the 2D pose predictions (twoDPose_HRN_train.npy and twoDPose_HRN_test.npy). You can make them smaller by converting to float32 and ignoring unused joints.

**Comment by wbhu** [comment]
Got it. Thanks very much.

**Comment by ZhangRenkai** [comment]
Thanks for your great work. I noticed that the 2D HRNet estimate step estimates 2D coordinates using cropped images (in the examples.zip file) , and I wonder where the cropped anchors come from, are they produced using an pretrained 2D detector or just generated by 2D GT labels. If an 2D detector is used, could you please provide the message or pretrained models download link of the detector model.
Thank you so much!!!
